File: Pattern/pattern3/Pattern.java
package Pattern.pattern3;

public class Pattern {
    public static void main(String[] args) {
        pattern(5);
    }

    public static void pattern(int n) {
        for (int i = 0; i <= n; i++) {
            for (int j = 0; j < i; j++) {
                System.out.print(i);
            }
            System.out.println("");
        }
    }
}

File: Pattern/pattern3/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := 0; i <= x; i++ {
		for j := 0; j < i; j++ {
			fmt.Print(i)
		}
		fmt.Println("")
	}
}

func main() {
	pattern(8)
}
